Core/Utilities/Profiler.py

```diff
@@ -0,0 +1,130 @@
+"""
+Profiling class for updated information on process status
+"""
+
+__RCSID__ = "$Id$"
+
+import psutil
```

Comment:
Note: psutil is not part of the standard library: took note in https://github.com/DIRACGrid/Externals/issues/11